tools/integration-tests/config.entry_node.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Having copies of config files here, means that we need to adapt everything on config changes or additions..
Is there a feasible way to avoid this? Like specifying only the changes in a json/flags?